.cicd/platforms/pinned/ubuntu-16.04-pinned.dockerfile

```diff
@@ -37,10 +37,12 @@ RUN git clone --depth 1 --single-branch --branch llvmorg-10.0.0 https://github.c
     make install && \
     cd / && \
     rm -rf /llvm
-# build boost
+# download Boost, apply fix for CVE-2016-9840 and build
 RUN curl -LO https://boostorg.jfrog.io/artifactory/main/release/1.72.0/source/boost_1_72_0.tar.bz2 && \
     tar -xjf boost_1_72_0.tar.bz2 && \
     cd boost_1_72_0 && \
+    curl -Lo boost/beast/zlib/detail/inflate_stream.ipp \
+    https://raw.githubusercontent.com/boostorg/beast/develop/include/boost/beast/zlib/detail/inflate_stream.ipp && \
```

Comment:
It's possible the version on master will change to be incompatible in the future. It's probably prudent to lock this download to a specific git commit

Reply:
Fixed in next commit